concent_api/core/tests/test_unit_message_handlers.py

```diff
@@ -156,3 +157,30 @@ def test_that_validation_raises_error_if_reason_is_none(self):
         )
         with self.assertRaises(GolemMessageValidationError):
             validate_reject_report_computed_task(incorrect_reject_reported_computed_task)
+
+
+class TestCorrectMessageToForcePaymentHandler(ConcentIntegrationTestCase):
+
+    def test_that_force_payment_without_subtask_results_accepted_list_raises_assertion(self):
+        force_payment = message.concents.ForcePayment()
+
+        with self.assertRaises(AssertionError):
+            handle_send_force_payment(force_payment)
+
+    def test_that_force_payment_with_empty_subtask_results_accepted_list_raises_assertion(self):
+        force_payment = message.concents.ForcePayment(subtask_results_accepted_list=[])
+
+        with self.assertRaises(AssertionError):
+            handle_send_force_payment(force_payment)
+
+    def test_that_force_payment_handler_will_raise_assertion_when_nested_message_has_wrong_type(self):
+        force_payment = message.concents.ForcePayment(subtask_results_accepted_list=[message.Ping()])
+
+        with self.assertRaises(AssertionError):
+            handle_send_force_payment(force_payment)
+
+    def test_that_force_payment_handler_will_raise_assertion_when_subtask_results_accepted_list_is_not_list_instance(self):
+        force_payment = message.concents.ForcePayment(subtask_results_accepted_list=())
+
+        with self.assertRaises(AssertionError):
+            handle_send_force_payment(force_payment)
```

Comment:
this can be simplified using pytest.parametrize. You can parametrize `subtask_results_accepted_list` parameter and write only one test. Now it is lot of code duplication

Reply:
Changed to `@pytest.mark.parametrize`